lib/symlink.go

```diff
@@ -93,21 +98,32 @@ func CheckSymlink(symlinkPath string) bool {
 // Deprecated: This function has been deprecated in favor of ChangeProductSymlink and will be removed in v2.0.0
 func ChangeSymlink(binVersionPath string, binPath string) {
 	product := getLegacyProduct()
-	ChangeProductSymlink(product, binVersionPath, binPath)
+	_ = ChangeProductSymlink(product, binVersionPath, binPath)
```

Comment:
Since it historically used log.Fatal in this method, since it was never capable of returning an error, in this version of the method, could we take the error and do a log.Fatal?